docs/issue_triage_guide.md

```diff
@@ -0,0 +1,39 @@
+# Issue Triage Guide
+
+This guide describes how to triage git issues on the [issue queue](https://github.com/openservicemesh/osm/issues) for Open Service Mesh (OSM).
+
+## Steps for Traige
+1. Identify the cause of the issue as explained below
+2. Tag the issue with relevant tags
+3. Assign the issue
+4. Prioritize the issue with a P1, P2 or P3 label where:
```

Comment:
@phillipgibson @bridgetkromhout How involved do you guys want to be as PMs in the prioritization process? Whatever we decide should be noted in this doc.

Reply:
Removed the prioritization section from the triage guide